core-api/src/test/java/com/optimizely/ab/internal/DefaultLRUCacheTest.java

```diff
@@ -0,0 +1,171 @@
+/**
+ *
+ *    Copyright 2022, Optimizely
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.internal;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class DefaultLRUCacheTest {
+
+    @Test
+    public void createSaveAndLookupOneItem() {
+        LRUCache<String> cache = new DefaultLRUCache<>();
+        assertNull(cache.lookup("key1"));
+        cache.save("key1", "value1");
+        assertEquals("value1", cache.lookup("key1"));
+    }
+
+    @Test
+    public void saveAndLookupMultipleItems() {
```

Comment:
This test looks good. It will be good to have the same ordering switch tests for save ops as well.

Reply:
Done!